actions/management/commands/fetch_action.py

```diff
@@ -45,13 +45,15 @@ def fetch_action(self, organisation, repo_name, **options):
 
         details = repo.get_repo_details()
         contributors = [i for i in repo.get_contributors() if "[bot]" not in i]
+        topics = sorted(repo.get_topics())  # order alphabetically for tidiness
```

Comment:
Let's not repeat ourselves: the code and the comment are almost the same, and "tidiness" is obvious enough.